python/ray/_private/test_utils.py

```diff
@@ -98,6 +98,18 @@ def redis_replicas():
     return int(os.environ.get("TEST_EXTERNAL_REDIS_REPLICAS", "1"))
 
 
+def enable_redis_sentinel():
+    import os
+
+    return os.environ.get("TEST_EXTERNAL_REDIS_SENTINEL") == "1"
+
+
+def redis_sentinel_replicas():
+    import os
+
+    return int(os.environ.get("TEST_EXTERNAL_REDIS_SENTINEL_REPLICAS", "3"))
```

Comment:
```suggestion
    return int(os.environ.get("TEST_EXTERNAL_REDIS_SENTINEL_REPLICAS", "2"))
```

nit: I think 2 is enough to test failover in the future.